docs/getting-started/install_capi_operator.md

```diff
@@ -2,6 +2,73 @@
 sidebar_position: 3
 ---
 
-# Install CAPI Operator
+# Install Cluster API Operator
 
-This section needs to be filled out.
\ No newline at end of file
+This section describes how to install `Cluster API Operator` in the Kubernetes cluster.
+
+## Installing CAPI and providers
+
+`CAPI` and desired `CAPI` providers could be installed using the helm-based installation for [`Cluster API Operator`](https://github.com/kubernetes-sigs/cluster-api-operator) or as a helm dependency for the `Rancher Turtles`.
+
+### Install as a `Rancher Turtles` dependency (recommended)
+
+See the `Rancher Turtles` section for installing the operator as a Helm [dependency](./install_turtles_operator.md#install-rancher-turtles-operator-with-cluster-api-operator-as-a-helm-dependency)
+
+### Install manually with Helm (alternative)
+To install `Cluster API Operator` with version `1.4.6` of the `CAPI` + `Docker` provider using helm, follow these steps:
```

Comment:
I think it is worth mentioning why we are using specific CAPI version, v1.4.6 in this case and later versions of it can be used in the future due to some limitations of the Rancher ?